src/components/Movies/Movies.jsx

```diff
@@ -0,0 +1,106 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { Link } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchMovies } from '../../redux/movies/movies';
+import Movie from './Movie';
+import YearsFilter from '../Filter/Filter';
+import '../../styling/Movies.scss';
+import '../../styling/Home.scss';
+
+const Movies = () => {
+  const moviess = useSelector((state) => state.movies);
+  const dispatch = useDispatch();
+  useEffect(() => { if (moviess.length === 0) { dispatch(fetchMovies('Home')); } }, []);
+
+  const moviesAll = Array.from(moviess).sort((a, b) => b.popularity - a.popularity);
+  const [filterTextValue, updateFilterText] = useState('All');
+
+  const movies = moviesAll.filter((movie) => {
+    if (filterTextValue === 'All') {
+      return movie;
+    }
+    return filterTextValue === movie.release_date.split('-')[0];
+  });
+
+  const ref = useRef(null);
```

Comment:
- [ OPTIONAL ] great job working on your project so far but I higly recommend making the navbar the same as the given design since it is very important to follow the design guidelines 💯✨ so kindly work on that to make it the same 👍🏻

- Your design 
![image](https://user-images.githubusercontent.com/58553711/205208129-f8763371-0e38-4cce-b4dd-a83034550f81.png)


- Given design
![image](https://user-images.githubusercontent.com/58553711/204962954-18497a5b-18ba-43bf-9b54-c6bccc0a4120.png)

Reply:
Thank you @Reem-lab I will work on the Navbar